rosboard/handlers.py

```diff
@@ -160,7 +160,7 @@ def on_message(self, message):
             topic_name = argv[1].get("topicName")
             max_update_rate = float(argv[1].get("maxUpdateRate", 0.2))            
             self.update_intervals_by_topic[topic_name] = 1.0 / max_update_rate
-            self.node.update_intervals_by_topic[topic_name] = max(
+            self.node.update_intervals_by_topic[topic_name] = min(
```

Comment:
@tommorel What was the reasoning for changing this to max? It chose min by default, but I see in your change you swapped it to select the max. The value is (1 / rate) so the **smaller** value between the user-selected and the default is really the **larger** update rate. Is this not what we want?